libs/Microsoft.MixedReality.WebRTC/RemoteAudioTrack.cs

```diff
@@ -35,6 +44,31 @@ public bool Enabled
         /// </summary>
         public event AudioFrameDelegate AudioFrameReady;
 
+        /// <summary>
+        /// Render the audio track to the system audio device.
+        /// </summary>
+        /// <remarks>
+        /// The default behavior is for every remote audio frame to be passed to
+        /// remote audio frame callbacks, as well as rendered automatically on the
+        /// system audio device. If |false| is passed to this function, remote audio
+        /// frames will still be received and passed to callbacks, but won't be rendered
+        /// on the system device.
+        ///
+        /// NOTE: Changing the default behavior is not supported on UWP.
+        /// </remarks>
+        public void RenderToDevice(bool render)
```

Comment:
nit: `OutputToDefaultDevice()`? I don't mind too much `Render`, but I'd like `Default` to appear somewhere for clarity, no?

Reply:
I actually do like "output" more than "render".

As discussed in chat, "Device" here means "whatever device WebRTC has decided to play tracks on", so "default" might be misleading here if WebRTC is configured not to use the default system device. Also to make things weirder WebRTC won't use the "default device" by default on Windows (!), but the "default communications device" which might be different. So let's avoid using "default" unless we can make the context clear.

Reply:
Changed the comments to talk about the "device used by WebRTC" for clarity.